cmd/derper/depaware.txt

```diff
@@ -180,6 +180,7 @@ tailscale.com/cmd/derper dependencies: (generated by github.com/tailscale/depawa
         bytes                                                        from bufio+
         compress/flate                                               from compress/gzip+
         compress/gzip                                                from internal/profile+
+   L    compress/zlib                                                from debug/elf
```

Comment:
These are due to a change in x/sys/unix (it depending on debug/elf now) and we need the newer x/sys/unix for wasip1 consts for josharian/native, etc.

Reply:
I sent https://go-review.googlesource.com/c/sys/+/502356 to fix upstream.